### PR TITLE
Submitting work done. Not finished :(

### PR DESCRIPTION
**Pull request by jonesandy**

### CODE REVIEW

File: lib/airport.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+class Airport
+  attr_reader :landed_planes
+  attr_reader :weather
+  attr_reader :capacity
+
+  def initialize
+    weather = Weather.new
+    @landed_planes = []
+    @weather = weather.report
```
Comment (ali-phipps) [inline lib/airport.rb]: nice I like the naming!

File: spec/airport_spec.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      allow(airport).to receive(:weather) { "sunny" }
+      airport.land(plane)
+      airport.takeoff(plane)
+      expect(airport.landed_planes).not_to include(plane)
+    end
+
+    it 'plane takes off' do
+      allow(airport).to receive(:weather) { "sunny" }
+      expect(airport.takeoff(plane)).to eq("Plane safely taken off.")
+    end
```
Comment (ali-phipps) [inline spec/airport_spec.rb]: I think you are missing a test to assert that your `plane.landed` attribute is set to `false` on takeoff

File: spec/airport_spec.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+require 'weather'
+
+describe Airport do
+  let(:plane) { Plane.new }
+  let(:airport) { Airport.new }
+
+  context 'landing a plane' do
+
+    it 'expects plane to be in airport' do
+      allow(airport).to receive(:weather) { "sunny" }
```
Comment (ali-phipps) [inline spec/airport_spec.rb]: DRY this with a before?

File: spec/airport_spec.rb [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      airport.land(plane)
+      expect(airport.landed_planes).to include(plane)
+    end
+
+    it 'checked that if plane already landed get error' do
+      allow(airport).to receive(:weather) { "sunny" }
```
Comment (ali-phipps) [inline spec/airport_spec.rb]: DRY with a `before`?

File: lib/airport.rb (hunk 1 above)
Comment (jonesandy) [reply lib/airport.rb]: Thank you

**Pull request closed without merging**